src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -202,7 +203,7 @@ private function createNode(DOMDocument $dom, string $nodeName): DOMElement
 
     private function addRandomTestsOrderAttributesIfNotSet(string $version, SafeDOMXPath $xPath): void
     {
-        if (!version_compare($version, '7.2', '>=')) {
+        if (version_compare($version, '7.2', '<')) {
```

Comment:
It seems like this change is missing a test. Or there is?..

Reply:
This method is covered by tests, I just simplified the condition:

https://github.com/infection/infection/blob/3aafcd43cec699620348cd5c40a197956308c37a/tests/phpunit/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilderTest.php#L410-L447